docs/sources/contribute-documentation/contribute-release-notes/index.md

```diff
@@ -97,32 +99,46 @@ Regardless of the status of your entry, it's always best to use the CMS to make
 
-If your entry is already live in both _What's new in Cloud_ and _What's new in Grafana `vMAJOR.MINOR.x`_, or it's after the cut-off date for a versioned release, update the CMS and then reach out to the person responsible for creating the versioned release notes.
+If your entry is already live in both _What's new in Cloud_ and it's between the cut-off date for a versioned release and the release date, update the CMS and then reach out to the person responsible for creating the versioned release notes.
+
+If your entry requires an update after it's live in both the Cloud and self-managed What's new, you'll need to update both entries.
 
 ### Create the versioned release notes
 
-1. After the previous version of Grafana is released, the directly responsible individual (DRI) cuts a branch and creates a draft PR with an empty What's new doc to be populated with the What's new content for the next release, along with the updated What's new index page. This PR:
+The following instructions are for the person directly responsible for creating the versioned release notes. This is typically someone on the Technical Writing team.
+
+1. After the cut-off date for What's new entries has passed, cut a branch and create a draft PR with an empty `whats-new-in-vxx-x.md` file to be populated with the What's new content for the next release. This PR should include:
+
+   - Updates to the `whatsnew/_index.md`
+   - Update to the link and version number located on the What's new tile of `docs/sources/_index.md`
+   - The new upgrade guide
+   - The new breaking changes page, if needed
+
+1. Label the PR `no-backport` for now; this may change.
+
+1. Have someone, typically the Tech Writing team build engineer, generate a markdown file from the _What's new in Cloud_ with the following conditions:
+
+   - Filtered by the relevant Grafana version
+   - Includes front matter for each entry
+   - Grouped by tags; entries with multiple tags should only be included once, grouped by their first tag alphabetically
 
-   - Should include an update to the link and version number located on the What's new tile of `docs/sources/_index.md`.
-   - Should include the new Upgrade Guide page.
-   - Should have a `no-backport` label.
-   - May include a new Breaking changes guide page.
+1. Add the content of this markdown file to the `whats-new-in-vxx-x.md` file using the tags data to group items.
 
-1. After the cut-off date for the self-managed/versioned release, the DRI collects the relevant What's new content (specifics TBD).
+   If there are internal enablement videos listed for entries, but the associated YouTube videos aren't in the body text of those entries yet, you'll need to add them later. To do this, generate another markdown file from the _What's new in Cloud_ closer to the release date and make updates in the `whats-new-in-vxx-x.md` file from it.
```

Comment:
```suggestion
   If internal enablement videos are listed for entries, but the associated YouTube videos aren't in the body text of those entries yet, you'll need to add them later. To do this, generate another markdown file from the _What's new in Cloud_ closer to the release date and make updates in `whats-new-in-vxx-x.md` from the newly generated file.
```

Small change around the internal enablements videos... 

The `from it` doesn't make sense to me... update from what? Are you making updates in the whats-new-in-vxx-x.md or in the newly generated file?

Reply:
You're making updates in the `whats-new-in-vxx-x.md`  file based on the newly generated What's new in Cloud file.